src/app/home/home.component.html

```diff
@@ -21,11 +22,21 @@
 </mat-menu>
 <!--Notification dropdown menu-->
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
-  <span mat-menu-item>Notification 1</span>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 2</div>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 3</div>
+  <span mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
+    <mat-divider></mat-divider>
+    <p class="menu-item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link">

```

Comment:
both read and unread message has same class. it same color will appera in both case